src/lib/utils.js

```diff
@@ -92,18 +92,35 @@ export function asyncMap (array, asyncCallback) {
 }
 
 /**
-  * Returns the currency symbol corresponding to the given currency.
-  *
-  * @param {string} currency - the currency to get the symbol for
-  */
+ * Returns a list of random keywords.
+ *
+ * @param {number} number
+ * @returns {Array<{value: string, hidden: boolean}>}
+ */
+export function generateRandomKeywords (number) {
```

Comment:
Sorry, I was not clear enough in my previous feedback. I didn't mean to move all the function to this `utils.js` file.
I was only talking about a `randomString` function: something like that:

```js
export function randomString(length) {
  return Math.random().toString(36).substring(2, length);
}
```

The `generateRandomKeyworks` function can be kept in the file you previously had because it is very specific to the component you are creating.

Sorry again for the confusion.